spec/models/order_item_spec.rb

```diff
@@ -6,7 +6,7 @@
     it {should validate_presence_of :item_id}
     it {should validate_presence_of :quantity}
     it {should validate_presence_of :ordered_price}
-    it {should validate_presence_of :fulfilled}
+    it {should validate_inclusion_of(:fulfilled).in_array([true,false])}
```

Comment:
Didn't know about this. I like how much more explicit it is on what should be there.